_data/product.yml

```diff
@@ -7,6 +7,7 @@ title_abbr_uc: MIQ
 appliance_rpm_dir: /opt/manageiq/manageiq-appliance
 operator_api_version: v1alpha1
```

Comment:
@bdunne should this be updated to reflect changes in https://github.com/ManageIQ/manageiq-pods/pull/778 ?

Reply:
No, this is the manageiq CRD version, not the global CRD API version if that makes sense.

Reply:
https://github.com/ManageIQ/manageiq-pods/blob/master/manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml